int16 loop_rate
---
bool  state
